components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
 <template>
-    <client-only>
-        <div class="map-container2">
-            <l-map
-                ref="map"
-                :zoom="zoom"
-                :bounds="localBounds"
-                :min-zoom="minZoom"
-                :max-zoom="21"
-                :max-bounds="maxBounds"
-                :max-bounds-viscosity="1"
-                :options="mapOptions"
-                @update:bounds="updateBounds"
+    <v-row no-gutters style="width: 1123px; height: 700px">
+        <v-col cols="8" class="pr-0">
+            <client-only>
+                <l-map
+                    ref="printMap"
+                    id="printMap"
+                    :zoom="zoom"
+                    :options="optionsMap"
+                    :bounds="bounds"
+                >
+                    <l-tile-layer
+                        url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+                        :attribution="attribution"
+                    ></l-tile-layer>
+
+                    <l-control-scale v-if="valueScale" position="bottomleft" />
+                    <l-control v-if="valueNorthArrow" position="bottomleft">
+                        <img
+                            src="../../assets/north-arrow.png"
```

Comment:
Check another way to add static files.

Reply:
Changed now nuxt will fetch the file (@)